weave/weave_client.py

```diff
@@ -625,6 +644,59 @@ def set_call_display_name(
     def remove_call_display_name(self, call: Call) -> None:
         self.set_call_display_name(call, None)
 
+    def feedback(
+        self,
+        query: Optional[Union[Query, str]] = None,
+        reaction: Optional[str] = None,
+        offset: int = 0,
+        limit: int = 100,
+    ) -> Feedbacks:
```

Comment:
Can you please add documentation to this method? I see two things that might be confusing for me:
1. `query` is implemented as `query_or_feedback_id`. I think you made this decision in order to make the first positional argument dual purpose and therefore nicer syntax. Anyway, would be good to make that clear
2. `reaction` -> is this supposed to be the literal emoji or shorthand? Both? Good to document